website/settings/defaults.py

```diff
@@ -88,10 +88,16 @@ def parent_dir(path):
 USE_EMAIL = True
 FROM_EMAIL = 'openscienceframework-noreply@osf.io'
 SUPPORT_EMAIL = 'support@osf.io'
+
+# SMTP Settings
 MAIL_SERVER = 'smtp.sendgrid.net'
 MAIL_USERNAME = 'osf-smtp'
 MAIL_PASSWORD = ''  # Set this in local.py
 
+# OR, if using Sendgrid's API
+SENDGRID_API_KEY = None
+
+
 # Mandrill
 MANDRILL_USERNAME = None
```

Comment:
Looks like they're just used in `scripts/admin_permission_email.py`. Makes me think that script is obsolete and should be removed.